platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -175,11 +184,53 @@ public void setUpTree() throws Exception {
     public void testInit() throws Exception {
         ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
                 testConfigAdminExt);
+        serviceBean.setMBeanServer(mBeanServer);
         serviceBean.init();
 
-        //Do it again, should remove and re-init
+        verify(mBeanServer).registerMBean(any(Object.class), any(ObjectName.class));
```

Comment:
Expectation should be `verify(mBeanServer).registerMBean(serviceBean, new ObjectName(ApplicationService.class.getName() + ":service=application-service")`. You want to verify that the right parameters are being passed to registerMBean() otherwise things won't work.